cmd/sealer/cmd/cluster/join.go

```diff
@@ -76,6 +76,9 @@ func NewJoinCmd() *cobra.Command {
 				return err
 			}
 
+			//store the Cluster as CfSnapshot for rollback
+			cf.CommitCfSnapshot(cf.GetCluster())
```

Comment:
"CommitCfSnapshot" => "CommitSnapshot".
And no need to pass the param "cf.GetCluster()".
Typically, I snapshot things based on current state of object.
Just "CommitCfSnapshot()"